frontend/src/styles/components/_buttons.scss

```diff
@@ -19,7 +18,7 @@ $btn-radius: 5px;
 
 // ----------------------------------------------------------------------
 .btn {
-  display: inline-flex;
+  //display: inline-flex;
```

Comment:
why was this changed?

Reply:
it removed the hyper link underline, i just put it back and fixed it in a better way